std_msgs/Header header
geometry_msgs/Pose pose
int32[] indices
float32[] w_p
float32[] s_p
float32[] si_q
float32[] si_q_theta_Rho
int32 timestep
